packages/gallery/src/components/gallery/proGallery/navigationPanel.js

```diff
@@ -96,6 +96,11 @@ class NavigationPanel extends React.Component {
                 style={itemStyle}
                 onClick={() => this.scrollToThumbnail(idx)}
                 tabIndex={0}
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter') {
```

Comment:
I think it's better to check with keyCode
also - we need to enter this code to V4 as well so it would be in TPA